### PR TITLE
feat: add columns API in Row

### PR DESCRIPTION
**Pull request by mrrtree**
# Which issue does this PR close?

Closes #

# Rationale for this change
 

Add some api in row for iteration, used in scenarios where the user does not know the schema

# What changes are included in this PR?

- Add struct `Column`
- `Row` add api `getColumns()` to get total columns in one row

# Are there any user-facing changes?

- `Row` getColumnValue(string) Value` is changed as `getColumn(string) Column`
- `Value` `Object Value()` is changed as `Object AnyValue()`

# How does this change test

All unit test is paased

### CODE REVIEW

File: ceresdb-protocol/src/main/java/io/ceresdb/models/Row.java [hunk 1]
```diff
@@ -3,39 +3,114 @@
...
+    public static class Column {
+        public Column(String name, Value value) {
+            this.name = name;
+            this.value = value;
+        }
+
+        private String name;
+        private Value  value;
+
+        public String name() {
```
Comment (MichaelLeeHZ) [inline ceresdb-protocol/src/main/java/io/ceresdb/models/Row.java]: How about `getName`?

File: ceresdb-protocol/src/main/java/io/ceresdb/models/Value.java [hunk 2]
```diff
@@ -72,7 +72,7 @@ public DataType getDataType() {
         return type;
     }
 
-    public Object getValue() {
+    public Object getAnyValue() {
```
Comment (ZuLiangWang) [inline ceresdb-protocol/src/main/java/io/ceresdb/models/Value.java]: I think `getValue()` is better than `getAnyValue()`, what does any mean?

Comment (MichaelLeeHZ) [inline ceresdb-protocol/src/main/java/io/ceresdb/models/Value.java]: I guess `getValue` or `getRawValue` is better?

Comment (mrrtree) [reply ceresdb-protocol/src/main/java/io/ceresdb/models/Value.java]: because row.getColumn("test").getValue().getValue() is confusing

Comment (mrrtree) [reply ceresdb-protocol/src/main/java/io/ceresdb/models/Value.java]: This is to be consistent with golang and rust

File: ceresdb-protocol/src/main/java/io/ceresdb/models/Row.java (hunk 1 above)
Comment (mrrtree) [reply ceresdb-protocol/src/main/java/io/ceresdb/models/Row.java]: ok

File: ceresdb-protocol/src/main/java/io/ceresdb/models/Value.java (hunk 2 above)
Comment (mrrtree) [reply ceresdb-protocol/src/main/java/io/ceresdb/models/Value.java]: rename to getObject()

**Review by MichaelLeeHZ — APPROVED** [review]
LGTM

**Review by ZuLiangWang — APPROVED** [review]
LGTM

**Pull request merged**